microsoft/testsuites/dpdk/dpdkutil.py

```diff
@@ -253,6 +253,7 @@ def initialize_node_resources(
 
     # netvsc pmd requires uio_hv_generic to be loaded before use
     if pmd == "netvsc":
+        node.mark_dirty()
```

Comment:
The "mark dirty" should be used after the main test logic, and the environment is dirty, cannot be reused. But here the dirty is used when initializing the environment. It won't affect the current test. Please add more comments, if it's intent.